### PR TITLE
JS Async Functions: API Status Challenge

### PR DESCRIPTION
**Pull request by PReichetanz**
rendered: https://github.com/neuefische/web-exercises/tree/feat/async-functions/api-status/sessions/js-async-functions/api-status

- [CSB](https://codesandbox.io/s/github/neuefische/web-exercises/tree/feat/async-functions/api-status/sessions/js-async-functions/api-status?file=/README.md)

refers to neuefische/web-curriculum-new-format#20

### CODE REVIEW

File: sessions/js-async-functions/api-status/js/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+const api = 'https://example-apis.vercel.app/api/status';
+
+async function checkIfAPIIsAvailable() {
+	const response = await fetch(api);
+}
+
+button.addEventListener('click', () => {
+	checkIfAPIIsAvailable();
+});
```
Comment (djfarly) [inline sessions/js-async-functions/api-status/js/index.js:12]:
```suggestion
// something needs to change here…
// ↙️
function checkIfApiIsAvailable() {
    /**
	 * Hint 1:
	 * Use the `fetch()` function and pass the `apiStatusUrl` into it
	 *
	 * Hint 2:
	 * The fetch function returns a promise which resolves to a Response
	 * object once it was received.
	 **/

	// --v-- write your code here --v--

	// --^-- write your code here --^--
}

button.addEventListener('click', () => {
	checkIfApiIsAvailable();
});
```

File: sessions/js-async-functions/api-status/js/index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const icon = document.querySelector('[data-js="status-icon"]');
+const button = document.querySelector('[data-js="status-button"]');
+
+const api = 'https://example-apis.vercel.app/api/status';
```
Comment (djfarly) [inline sessions/js-async-functions/api-status/js/index.js:4]:
```suggestion
const apiStatusUrl = 'https://example-apis.vercel.app/api/status';
```

**Review by djfarly — CHANGES_REQUESTED** [review]
this is not a full review 🫠

**Comment by CJAkkuec** [comment]
Hey there, this was very fun - as per usual! :) Here's my (maybe a little whacky) [solution](https://codesandbox.io/s/crimson-river-wxfb0q?file=/js/index.js). No notes, all good imho.

**Review by CJAkkuec — APPROVED** [review]

**Pull request merged**